observability/network_observability/network-observability-secondary-networks.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="network-observability-secondary-networks"]
+= Secondary networks
+include::_attributes/common-attributes.adoc[]
+:context: secondary_networks
```

Comment:
This should normally match the ID/file name

```suggestion
:context: network-observability-secondary-networks
```

Remember to change the xrefs that reference this if you fix it

Reply:
Really? None of the contexts in network_observability assemblies match the filename 😬 I do see this now in the code callout of the contributing guide. Thanks for pointing it out.